src/application.ts

```diff
@@ -34,6 +35,20 @@ export class Application {
     this.router = opts.router || express.Router() // you can do this?
   }
 
+  /**
+   * Loads a Probot plugin
+   * @param {function} plugin - Probot plugin to load
+   */
+  public load (app: Plugin | Plugin[]) : Application {
+    if (Array.isArray(app)) {
```

Comment:
I added this additional use case:

```js
app.load([
  require('./plugin'),
  require('./another-plugin')
])
```

If for some reason you feel that this shouldn't exist, lemme know, I'm not married to it but I do have a use-case for it in a project of mine.